quarkus/server/distribution/LICENSE

```diff
@@ -0,0 +1,2482 @@
+
+--------------------------------------------------------------------------------
+
+Group: com.azure Name: azure-core Version: 1.55.2
+Project URL (from POM): https://github.com/Azure/azure-sdk-for-java
+License (from POM): The MIT License (MIT) - http://opensource.org/licenses/MIT
+
+--------------------------------------------------------------------------------
+
+Group: com.azure Name: azure-core-http-netty Version: 1.15.10
+Project URL (from POM): https://github.com/Azure/azure-sdk-for-java
+License (from POM): The MIT License (MIT) - http://opensource.org/licenses/MIT
+
+--------------------------------------------------------------------------------
+
+Group: com.azure Name: azure-identity Version: 1.15.3
+Project URL (from POM): https://github.com/Azure/azure-sdk-for-java
+License (from POM): The MIT License (MIT) - http://opensource.org/licenses/MIT
+
+--------------------------------------------------------------------------------
+
+Group: com.azure Name: azure-json Version: 1.4.0
+Project URL (from POM): https://github.com/Azure/azure-sdk-for-java
+License (from POM): The MIT License (MIT) - http://opensource.org/licenses/MIT
+
+--------------------------------------------------------------------------------
+
+Group: com.azure Name: azure-storage-blob Version: 12.29.1
+Project URL (from POM): https://github.com/Azure/azure-sdk-for-java
+License (from POM): The MIT License (MIT) - http://opensource.org/licenses/MIT
+
+--------------------------------------------------------------------------------
+
+Group: com.azure Name: azure-storage-common Version: 12.28.1
+Project URL (from POM): https://github.com/Azure/azure-sdk-for-java
+License (from POM): The MIT License (MIT) - http://opensource.org/licenses/MIT
+
+--------------------------------------------------------------------------------
+
+Group: com.azure Name: azure-storage-file-datalake Version: 12.22.1
+Project URL (from POM): https://github.com/Azure/azure-sdk-for-java
+License (from POM): The MIT License (MIT) - http://opensource.org/licenses/MIT
+
+--------------------------------------------------------------------------------
+
+Group: com.azure Name: azure-storage-internal-avro Version: 12.14.1
+Project URL (from POM): https://github.com/Azure/azure-sdk-for-java
+License (from POM): The MIT License (MIT) - http://opensource.org/licenses/MIT
+
+--------------------------------------------------------------------------------
+
+Group: com.azure Name: azure-xml Version: 1.2.0
+Project URL (from POM): https://github.com/Azure/azure-sdk-for-java
+License (from POM): The MIT License (MIT) - http://opensource.org/licenses/MIT
+
+--------------------------------------------------------------------------------
+
+Group: com.fasterxml Name: classmate Version: 1.7.0
+Project URL (from POM): https://github.com/FasterXML/java-classmate
+License (from POM): Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
```

Comment:
Question: is there a tool to generate these or is this done by manually?

Reply:
It's a mix: I'm using a tool to assist on the `LICENSE` generation, but the "final" result is verified manually and `NOTICE` is by hand (as it depends of the ALv2 license and provided `NOTICE` by dependent projects).